flexget/plugins/filter/unique.py

```diff
@@ -0,0 +1,92 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+
+log = logging.getLogger('unique')
+
+
+class Unique(object):
+    """
+    Take action on entries with duplicate fields, except for the first item
+
+    Reject the second+ instance of every movie:
+
+      unique:
+        field:
+          - imdb_id
+          - movie_name
+        action: reject
+    """
+
+    schema = {
+        'oneOf': [
+            {'type': 'boolean'},
+            {
+                'type': 'object',
+                'properties': {
+                    'field': {
+                        'oneOf': [
+                            {'type': 'string'},
+                            {'type': 'array'},
+                        ]
+                    },
+                    'action': {'enum': ['accept', 'reject']},
+                },
+                'required': [],
```

Comment:
I copied this behavior from the "duplicates" plugin. I think it's a good behavior for ensuring uniqueness of simple entities.
Would you have required specifying `field`?